src/components/HotelDetails/HotelDetails.jsx

```diff
@@ -0,0 +1,144 @@
+  };
+  return (
+    <div className="card_details">
+      <div className="marquee">
+        <div className="marquee-content">
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[0]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[1]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[2]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[3]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[0]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[1]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[2]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[3]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[0]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[1]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[2]}
+              alt={hotel.name}
+            />
+          </div>
+          <div className="marquee-item">
+            <img
+              className="img-details"
+              src={hotel.photo[3]}
+              alt={hotel.name}
+            />
+          </div>
+        </div>
+        {/* <img className="img-details" src={hotel.photo[0]} alt={hotel.name} />
+          <img className="img-details" src={hotel.photo[1]} alt={hotel.name} />
+          <img className="img-details" src={hotel.photo[2]} alt={hotel.name} />
+          <img className="img-details" src={hotel.photo[3]} alt={hotel.name} />
+          <FaChevronLeft className="left-arrow" />
+          <FaChevronRight className="right-arrow" /> */}
```

Comment:
- [ ] Kindly remove this old piece of commented code to improve the repo's cleanliness. 👍